web/css/libs/prism.css

```diff
@@ -7,76 +7,76 @@
 
 code[class*="language-"],
 pre[class*="language-"] {
-	color: black;
-	text-shadow: 0 1px white;
-	font-family: Consolas, Monaco, 'Andale Mono', monospace;
-	direction: ltr;
-	text-align: left;
-	white-space: pre;
-	word-spacing: normal;
-	word-break: normal;
+    color: black;
```

Comment:
please revert any change to these `libs` files. these are third-party files.

Reply:
Whoops! Done.